lib/components/viewers/route-viewer.js

```diff
@@ -8,9 +8,17 @@ import styled from 'styled-components'
 import { FormattedMessage, injectIntl } from 'react-intl'
 
 import Icon from '../util/icon'
-import { setMainPanelContent, setViewedRoute, setRouteViewerFilter } from '../../actions/ui'
+import {
+  setMainPanelContent,
+  setViewedRoute,
+  setRouteViewerFilter
+} from '../../actions/ui'
 import { findRoutes, findRoute } from '../../actions/api'
-import { getSortedFilteredRoutes, getModesForActiveAgencyFilter, getAgenciesFromRoutes } from '../../util/state'
+import {
+  getSortedFilteredRoutes,
+  getModesForActiveAgencyFilter,
+  getAgenciesFromRoutes
```

Comment:
Move this up (a-z)

Reply:
Done in 59bdf8f495ea26a3a31f7b48ad10a767b0ca8ef7